src/App/Fossa/Container/AnalyzeNative.hs

```diff
@@ -87,10 +94,35 @@ parseContainerImageSource ::
   ( Has (Lift IO) sig m
   , Has Diagnostics sig m
   , Has ReadFS sig m
+  , Has Logger sig m
   ) =>
   Text ->
   m (ContainerImageSource)
-parseContainerImageSource = parseExportedTarballSource
+parseContainerImageSource src =
+  parseExportedTarballSource src
+    <||> parseDockerEngineSource src
+
+parseDockerEngineSource ::
```

Comment:
- [ ] Add Unit test for `parseDockerEngineSource` by mocking `runDockerEngineApi`